discord_bot/discord_bot.py

```diff
@@ -119,13 +127,13 @@ def get_gateway(self):
                 elif message["op"] == Opcodes.HEARTBEAT_ACK:
                     pass
                 elif message["op"] == Opcodes.INVALID_SESSION:
-                    print("Invalid Session")
+                    self.logger.info("Invalid Session")
                 elif message["op"] == Opcodes.DISPATCH:
                     event = message["t"]
                     if event == Events.READY:
                         self.session_id = message["d"]["session_id"]
                 else:
-                    print(message)
+                    self.logger.info(message)
```

Comment:
Eventually this `else` case should be catching unexpected values, so this should probably be `warning` level at least. Probably want to include additional info with the message, such as

```self.logger.exception("Unexpected opcode {}: {}".format(message["op"], message))```